docs/building-images.md

```diff
@@ -157,7 +157,46 @@ operatingSystem:
   * `additionalRepos` - Optional; List of third-party RPM repositories that will be added to the package manager of the OS.
   * `sccRegistrationCode` - Optional; SUSE Customer Center registration code, used to connect to SUSE's internal RPM repositories.
 
-## SUSE Manager (SUMA)
+### Kubernetes
```

Comment:
```suggestion
### Kubernetes (K3s or RKE2)
```

Reply:
I'll add a note that only these two distros are supported but I wouldn't want to specify it here since I'm referring to the specific section in the definition file.